workshop/session_3/11_biopython_proteins.ipynb

```diff
@@ -0,0 +1,957 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 162,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Count of particular aminoacids\n",
+      "{'A': 83, 'C': 18, 'D': 58, 'E': 93, 'F': 85, 'G': 84, 'H': 25, 'I': 119, 'K': 92, 'L': 183, 'M': 37, 'N': 54, 'P': 45, 'Q': 67, 'R': 78, 'S': 123, 'T': 83, 'V': 90, 'W': 23, 'Y': 40}\n",
+      "Using PrettyPrint for more user-friendly representation.\n",
+      "{'A': 83,\n",
+      " 'C': 18,\n",
+      " 'D': 58,\n",
+      " 'E': 93,\n",
+      " 'F': 85,\n",
+      " 'G': 84,\n",
+      " 'H': 25,\n",
+      " 'I': 119,\n",
+      " 'K': 92,\n",
+      " 'L': 183,\n",
+      " 'M': 37,\n",
+      " 'N': 54,\n",
+      " 'P': 45,\n",
+      " 'Q': 67,\n",
+      " 'R': 78,\n",
+      " 'S': 123,\n",
+      " 'T': 83,\n",
+      " 'V': 90,\n",
+      " 'W': 23,\n",
+      " 'Y': 40}\n"
+     ]
+    }
+   ],
+   "source": [
+    "count_of_aas = analysis.count_amino_acids() \n",
+    "print(\"Count of particular aminoacids\")\n",
+    "print(count_of_aas)\n",
+    "print(\"Using PrettyPrint for more user-friendly representation.\")\n",
+    "import pprint\n",
+    "pprint.pprint(count_of_aas)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": false
+   },
+   "source": [
+    "Let's take a look at some protein properties available.\n",
+    "\n",
+    "`\"{:.2f}\"` is used to print only a `float` number to the first two decimal places."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 163,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Molecular weigth    : 168139.53\n",
+      "Charge at a given pH: 38.97\n",
+      "Isoelectric point   : 8.91\n",
+      "Fractions of AA associated with secondary structures:\n",
+      "  Helix: 0.36\n",
+      "  Turn: 0.21\n",
+      "  Sheet: 0.27\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Molecular weigth    :\", \"{:.2f}\".format(analysis.molecular_weight()))\n",
+    "print(\"Charge at a given pH:\", \"{:.2f}\".format(analysis.charge_at_pH(5.8)))\n",
+    "print(\"Isoelectric point   :\", \"{:.2f}\".format(analysis.isoelectric_point()))\n",
+    "in_helix, in_turn, in_sheet = analysis.secondary_structure_fraction()\n",
+    "print(\n",
+    "    \"Fractions of AA associated with secondary structures:\\n\"\\\n",
+    "    \"  Helix: {:.2f}\\n\"\\\n",
+    "    \"  Turn: {:.2f}\\n\"\\\n",
+    "    \"  Sheet: {:.2f}\\n\".format(in_helix, in_turn, in_sheet)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "editable": false
+   },
+   "source": [
+    "We can also use the helper functions provided to create new statistics. \n",
+    "\n",
+    "For example, let's calculate BCAA (branch-chain amino acid) content of the protein."
```

Comment:
lets add a link for BCAA (https://en.wikipedia.org/wiki/Branched-chain_amino_acid)